### ISSUE TITLE
Export

### ISSUE THREAD

**Issue by twikedk** [renyuneyun/Easer#146]
How is export suppose to work?? If I hit the button, there is no information and I have no clue where to find the zip file.

If I look under app info for Easer it says:
Permissions
This app can access the following on your phone:
* modify or delete the contents of your SD card
* read the content of your SD card

I'm not sure if there should be a point saying directly write but else I would guess it should work by permissions.

Easer v0.6.7.2
Android 5.1

Added log information:
And the log file doesn't give much except that it apparently has succeeded or at least not failed.

[logs_0.csv.txt](https://github.com/renyuneyun/Easer/files/2214230/logs_0.csv.txt)

But if I look in /storage/sdcard1/ for Easer.*.zip files there are none.

It can't be the naming that Android dislikes?

**Comment by renyuneyun** [comment]
Hmm, strange...

The exported file should be located at `/sdcard/Easer.YEAR-MONTH-DATE.HOUR:MINUTE:SECOND.zip` (where `/sdcard` may be a symlink to `/storage/SOMETHING`).
The name is normal for unix (and so for Android, because of linux kernel).

Could you check if `/sdcard` on your device the same as `/storage/sdcard1`?

**Comment by twikedk** [comment]
if it is the same place /logger goes then it should be /storage/sdcard1 but it looks like /sdcard symlinks to /storage/emulated/legacy which also doesn't contain any Easer.*.zip file and I have checked in the folders /sdcard0 (phone) and /sdcard1 (ext. SD card) none holds a Easer.*.zip file.

I noticed that if I hit import I'm asked where to load from maybe something similar could be done with the export function? This also would be a clue that something is happening.
